src/main/java/frc/robot/SwerveModule.java

```diff
@@ -15,66 +10,66 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.kinematics.SwerveModulePosition;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
-import edu.wpi.first.wpilibj.AnalogEncoder;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import frc.robot.constants.SwerveConstants;
 
 public class SwerveModule {
     // Parameters
-    private int _rotID;
-    private int _transID;
-    private int _rotEncoderID;
-    private boolean _rotInverse;
-    private boolean _transInverse;
-    private PIDController _rotPID;
+    private int mRotID;
```

Comment:
Several of these fields can be made final.

Reply:
Done